model/src/main/kotlin/spdx/SpdxExpression.kt

```diff
@@ -46,10 +46,16 @@ data class SpdxLicenseRefExpression(
         val id: String
 ) : SpdxExpression()
 
-enum class SpdxOperator {
-    AND,
-    OR,
-    WITH
+enum class SpdxOperator(
+        /**
+         * The priority of the operator. An operator with a higher priority binds stronger than an operator with a lower
```

Comment:
Should we say "larger priority value" here to make it super-clear that by "higher" (in the original text) we do not refer to "priority" but to the integer value? Because in some systems 0 actually is the ultimate highest priority.

Reply:
Done.